service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseDataExtractionService.java

```diff
@@ -237,12 +237,19 @@ private DocmosisChild buildChild(ChildParty child) {
             .build();
     }
 
-    private LocalDateTime calculatePrehearingAttendance(LocalDateTime dateTime) {
-        return dateTime.minusHours(1);
-    }
+    public String getHearingAttendance(HearingBooking hearingBooking) {
+
+        if (isEmpty(hearingBooking.getAttendance())) {
```

Comment:
I'd use Optional here or extract both `hearingBooking.getAttendance()` into a variable. Just personal opinion, though.

Reply:
changed, but i think code is a bit more complicated